pkg/cmd/roachtest/tests/mixed_version_cdc.go

```diff
@@ -40,7 +42,11 @@ const (
 
 	// resolvedInterval is the value passed to the `resolved` option
 	// when creating the changefeed
-	resolvedInterval = "10s"
+	resolvedInterval = "5s"
+
+	// kafkaBufferMessageSize is the number of messages from kafka
+	// we allow to be buffered in memory before validating them.
+	kafkaBufferMessageSize = 8192
```

Comment:
I wonder if this will lead to flakes. I remember that when I wrote the initial version of this mixed-version cdc test, we were seeing failures because CDC was emitting thousands of old events when nodes restarted (#89332). If that were to happen again, I can see the buffer overgrowing this limit even though there's no bug per se?